.gitignore

```diff
@@ -5,5 +5,4 @@ npm-debug.log
 .ds_store
 dist
 veracode
-.size-snapshot.json
-reports
```

Comment:
Hmm, removing reports from `.gitignore` might unintentionally bring the `eslint.xml` into version control when running `npm run lint:junit`. Should we exclude `*eslint.xml` instead?